demo/__tests__/App.test.tsx

```diff
@@ -17,7 +17,13 @@ describe('App', () => {
 
     // Open http://localhost:3336 to see preview
     // Require to run `jest-preview` server before
-    preview.debug();
-    expect(screen.getByTestId('count')).toContainHTML('6');
+    // Execute `preview.debug()` or `debug()` to see the UI in a browser
+    debug();
+
+    // Jest Preview automatically preview failed tests without explicitly calling `debug()`
+    // Try to uncomment the following line to see the count equals to 6
```

Comment:
I think what you are trying to say is `Try to comment out`?

Reply:
Thanks for catching that.